schema/badges.ts

```diff
@@ -0,0 +1,51 @@
+import { z } from 'zod';
+import { dateSchema } from '../util/sharedSchema';
+
+export const VERSION = '1.0.0';
+
+export const schema = z
+  .object({
+    badgeId: z.string(),
+    name: z.string(),
+    displayName: z.string(),
+    description: z.string(),
+    link: z.string(),
+    icon: z.string(),
```

Comment:
Nit: how about using [`z.string().url()`](https://zod.dev/?id=strings) so that it's clear we should put URLs inside?